shared/protocol/src/test/java/io/pravega/shared/protocol/netty/AppendEncodeDecodeTest.java

```diff
@@ -399,6 +410,28 @@ private void testFlush(int size) throws Exception {
         assertEquals(keepAlive, two);
     }
 
+
+    @Test
+    public void testAppendWithoutBatchTracker() throws Exception {
```

Comment:
I'm not sure what it means "without batch tracker" in the context of this test case.

Reply:
if there is no append batch size tracker in the CommandEncoder then every append is followed by the appendblockend. No, blockTimeout  timer configuration.